leo/package/src/lib.rs

```diff
@@ -184,6 +187,13 @@ pub fn fetch_program_from_network(name: &str, endpoint: &str, network: NetworkNa
 pub fn verify_valid_program(name: &str, program: &str) -> Result<(), UtilError> {
     use snarkvm::prelude::{Program, TestnetV0};
     use std::str::FromStr as _;
+    
+    let program_size = program.len();
```

Comment:
Please add a comment at each logical block, even if it may be obvious. It's our coding style.